docs/guides/admin/docs/configuration/security.user.brightspace.md

```diff
@@ -8,9 +8,10 @@ member of the Brightspace course `myCourseID` will be granted the Opencast role
 
 ### Requirements
 
+<!-- _This section has been left blank! It should be filled with the necessary info, or removed entirely! -->
```

Comment:
File an issue for this please. Assign it to me and I'll track down the Brightspace folks.

Reply:
issue opened!